data/obs-packaging/clear-containers-image/README.md

```diff
@@ -0,0 +1,18 @@
+#clear-containers-image
+
+This directory contains the sources to update rpm specfiles and debian source
+control files to generate ``clear-containers-image`` packages for Fedora and Ubuntu.
+
+There is a script to generate specs and dsc automatically, to be pushed to
+Open Build Service (http://openbuildservice.org/) and generate packages for you.
+
+``./update_image.sh``
+
+This script will generate/update:
```

Comment:
Avoid using slashes, they are ambiguous. Do you mean:
This script generates and updates:
OR
This script generates or updates: